chingu-coders/Voyage2-Turtles-02
# attempt to fix CSS validator URI content bug

## Purpose
* CSS Validator was returning buggy results.

## Approach/Fixes
* I suspect the problem was that the value of the `textarea` was not being properly formatted to be sent over URI query. I used `encodeURIComponent` to properly format the value.

## Testing
* checkout this PR.
* add some valid and invalid CSS to the validator and check that the error responses are valid
